code/modules/power/solar.dm

```diff
@@ -198,6 +198,14 @@
 		S.amount = 2
 		glass_type = null
 
+/obj/item/solar_assembly/examine(mob/user)
+	. = ..()
+	. += "<span class='notice'>The solar assembly is <b>[anchored ? "wrenched into place" : "unwrenched"]</b>.</span>"
+	if(tracker)
+		. += "<span class='notice'>The solar assembly has a tracking circuit installed. It can be <b>pried out</b>.</span>"
+	else
+		. += "<span class='notice'>The solar assembly has a slot for a tracking circuit board.</span>"
+	.+= "<span class='notice'>The solar assembly needs <i>glass<i> to be completed.</span>"
```

Comment:
This should only show up if the assembly is anchored, yeah? That should probably be changed